.gitignore

```diff
@@ -4,5 +4,5 @@ node_modules
 npm-debug.log
 /.idea
 .tern-port
-lib
 build
+lib
```

Comment:
Nah, I'm just particular and everything else was styleguide then library, nbd though